README.md

```diff
@@ -12,22 +12,34 @@ Highly Available Tomcat 9 Image
        -> Update your application build.yaml to use this image for hosting your java application 
        -> Update your application deployment.yaml to add the rbac authorizations and roles required to run the tomcat in a cluster and don't forget to add the service account required to run your container under the spec.
 
-#### Note: To run with existing docker configuration, you will not be needed to make any changes to the docker folder
+#### Note: To run with existing docker configuration, you will not be need to make any changes to the docker folder
 
-# How to build image
-## In Openshift
+# Building & Deploying the Image
+## In Openshift (via developer CLI)
 ```
+We tag images using the name of the Jira Issue that caused the image to be built (the SUFFIX parameter).
+Per openshift requirements, we do this in lower-case.
+If you are just running an experiment, you can use your name, your idir, or 'build'.
```

Comment:
So which command is this pertaining too? Taking a hint from nr-elasticsearch documentation, It may be worthwhile to say 
"for e.g. <this command> and idir , john."
I mean please provide an example of the command when someone is just running an experiment.

Reply:
Rearranged to show the SUFFIX parameter, and then explain its uses.